rust/benchmarks/benches/interpreter.rs

```diff
@@ -6,8 +6,8 @@ use benchmarks::RunArgs;
 use criterion::{criterion_group, criterion_main, Criterion};
 
 fn criterion_benchmark(c: &mut Criterion) {
-    let args = RunArgs::ffi_overhead();
-    c.bench_function("ffi_overhead", |b| b.iter(|| benchmarks::run(&args)));
```

Comment:
Because args is now mutable, best benchmarking would be to have a fresh instance per iteration (truly stateless) 

This is a common pattern when benchmarking, 

```
static context init
Benchmark loop {
      Off timer section: transient state construction
      On Timer secition: the code benchmarked
}


Is this posible using criterion?

Reply:
yes, the example presented is a common pattern which achieve two goals: it keeps stateless runs (no carry over data)  but measures time only for the critical path. 

see https://github.com/google/benchmark/blob/main/docs/user_guide.md pause function.

Reply:
I see. But it seems like criterion has nothing like that.